### PR TITLE
feat(domain): add possibility to have a custom domain

### PR DESCRIPTION
**Pull request by javierpaez**
### Checklist

- [ ] All new jobs, commands, executors, parameters have descriptions
- [ ] Examples have been added for any significant new features
- [ ] README has been updated, if necessary

### Motivation, issues
In some organizations there is another instance of CCI, we need to have a custom domain and I added a new env variable called CIRCLECI_DOMAIN to set the custom host

### Description
change the circleci.com to custom domain using $CIRCLECI_DOMAIN

### CODE REVIEW

File: src/jobs/workflow-collector.yml [hunk 1]
```diff
@@ -3,20 +3,20 @@ description: >
 parameters:
   workflow-collector:
     type: env_var_name
-    default: "WORKFLOW_HTTP_SOURCE"
-    description: "Enter the HTTP Source endpoint generated by the Sumologic setup wizard for Workflows."
+    default: 'WORKFLOW_HTTP_SOURCE'
```
Comment (gmemstr) [inline src/jobs/workflow-collector.yml]: Let's revert the change from double to single quotes to clean up this PR and stay in line with the rest of the orb.

**Review by gmemstr — CHANGES_REQUESTED** [review]

File: src/jobs/workflow-collector.yml (hunk 1 above)
Comment (javierpaez) [reply src/jobs/workflow-collector.yml]: done @gmemstr

**Comment by KyleTryon** [comment]
Hello, 

Thank you for your contributions. We have connected with SumoLogic to take over ownership of this orb. We will be archiving this repository shortly.

The new SumoLogic orb and repository can be found here:
 - repo: https://github.com/SumoLogic/sumologic-orb
 - orb: https://circleci.com/developer/orbs/orb/sumologic/sumologic

Ensure to update the orb reference in your config with the orb now under the SumoLogic namespace.

```
orbs:
  sumologic: sumologic/sumologic@2.1.0
```
